blog/2022/01/22/o2-orion-monopoles.mdx

```diff
@@ -0,0 +1,173 @@
+
+# Planning applications
+
+For mobile networks to install new street infrastructure, they do not need to apply for planning permission. Instead, they must submit a notice
+of **prior notification**. This is a notice of intent, basically saying "I want to do something. Tell me if I can't.".
+
+The local planning authority has **56 days to object to the plans, or approve them**. No response after 56 days is classed as an approval. The
+big kick against conspiracy theorists is that **LPAs may only reject proposed development due to its siting or appearance**.
+
+The compact nature of Orion monopole sites mean that councils are more likely to approve them compared to monopoles from other operators which
+may clutter public areas with various street furniture.
+
+## Exemptions
+
+Exemptions exist from submitting prior notification plans, aimed to make minor changes easier for network operators. While they are meant for
+swapping out equipment when it fails or for adding new microwave link dishes to sites, O2 have now been seen using these powers for performing
+full-on Orion site swaps in some areas.
+
+One of my local monopole sites in Burgess Hill, for example, has recently been swapped from a Beacon 2 site to a new dual operator Orion with no
+prior notice submitted. This was possible due to one exemption:
+
+> ...prior approval does not apply to ground-based masts which when altered or replaced would not exceed the greater of the height of the
+> existing mast and 20m above ground level.
+
+Since my old local monopole was 20m in height, and so was the new Orion monopole, **the site could be swapped out without any form of approval
+required**.
+
+# Orion equipment
+
+Orion installations differ between vendor zones. The majority of sites we've seen have been O2 <Tooltip text="only O2">unilaterals</Tooltip>
+using Ericsson equipment. We've seen some Cornerstone shared sites (where O2 and Vodafone are both located on the monopole), though.
+
+## Nokia vendor
+
+[My site](https://www.google.com/maps/place/50%C2%B058'00.2%22N+0%C2%B008'13.7%22W/@50.9667335,-0.1376914,19z) was one of the first we'd seen in
+the wild using Nokia equipment. It was also a dual-operator site, with O2 and Vodafone both broadcasting from it.
+
+Unlike the [previous Ericsson sites seen by Jake and Peter in Cardiff](https://www.youtube.com/watch?v=ou_CGjbQ8uc), this site doesn't appear to
+be kitted out for a massive MIMO deployment, instead preferring 8T8R for n78 5G.
+
+On the top of the monopole are three
+[Commscope 24-port RRZZT4S4-65B-R6 antennas](https://www.commscope.com/product-type/antennas/base-station-antennas-equipment/base-station-antennas/itemrrzzt4s4-65b-r6/)
+(that's a mouthful), each fully loaded with feeder cables. This is different from the Huawei passive antennas we had previously seen in Ericsson
+zones.
+
+The street furniture used was a single Yorkshire cabinet, while the previous three cabinets (one for each operator, plus one for power) were
```

Comment:
```suggestion
The street furniture used was a single Yorkshire cabinet, while the previous three cabinets (one for each operator, plus one for power) were removed as part of the upgrade on day four.
```
Numbers ten and under should be spelled out, day isn't a proper noun in this context.